modules/features/kids/src/main/java/au/com/shiftyjelly/pocketcasts/kids/KidsBanner.kt

```diff
@@ -0,0 +1,149 @@
+import au.com.shiftyjelly.pocketcasts.localization.R.string
+import au.com.shiftyjelly.pocketcasts.localization.R.string.close
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+
+@Composable
+fun KidsProfileCard(
+    onDismiss: () -> Unit,
+) {
+    val configuration = LocalConfiguration.current
+    val isLandscape = configuration.orientation == Configuration.ORIENTATION_LANDSCAPE
+    val bannerContentDescription = LocalContext.current.getString(string.kids_profile_banner)
+
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp)
+            .background(MaterialTheme.theme.colors.primaryUi01Active, shape = RoundedCornerShape(8.dp))
+            .semantics { this.contentDescription = bannerContentDescription },
+    ) {
+        Image(
+            colorFilter = ColorFilter.tint(MaterialTheme.theme.colors.primaryUi05Selected),
+            painter = painterResource(id = ic_close),
+            contentDescription = stringResource(id = close),
+            modifier = Modifier
+                .padding(11.dp)
+                .size(22.dp)
+                .align(Alignment.TopEnd)
+                .clickable { onDismiss() },
+        )
+
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Column(
+                modifier = Modifier.weight(1f),
+            ) {
+                Row(
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.spacedBy(6.dp),
+                    modifier = Modifier.padding(start = 24.dp, top = 16.dp, bottom = 8.dp),
+                ) {
+                    TextH40(
+                        text = stringResource(string.kids_profile),
+                        fontWeight = FontWeight.W500,
+                    )
+                    Box(
+                        contentAlignment = Alignment.Center,
+                        modifier = Modifier
+                            .background(MaterialTheme.theme.colors.primaryField01Active, shape = RoundedCornerShape(4.dp))
+                            .width(40.dp)
+                            .height(24.dp),
+                    ) {
+                        Text(
+                            text = stringResource(string.soon_label),
+                            color = MaterialTheme.theme.colors.primaryUi05Selected,
+                            fontSize = 11.sp,
+                            fontWeight = FontWeight.W600,
+                        )
+                    }
+                }
+                TextH70(
+                    text = stringResource(string.kids_profile_banner_description),
+                    color = MaterialTheme.theme.colors.primaryUi05Selected,
+                    modifier = Modifier
+                        .padding(start = 24.dp, bottom = 8.dp)
+                        .then(if (isLandscape) Modifier else Modifier.width(200.dp)),
```

Comment:
Instead of doing a different version for a landscape layout it wouldn't be simpler to and an end padding equal to the image's width (`116.dp`). This way text will always use maximum available space regardless of the screen size.

Reply:
I liked that!